crates/stages/stages/src/stages/hashing_storage.rs

```diff
@@ -101,9 +101,15 @@ where
                 let chunk = chunk.collect::<Result<Vec<_>, _>>()?;
                 // Spawn the hashing task onto the global rayon pool
                 rayon::spawn(move || {
+                    // Cache hashed address since PlainStorageState is sorted by address
+                    let (mut last_addr, mut hashed_addr) = (Address::ZERO, B256::ZERO);
```

Comment:
Even though it will basically never matter, might as well make this use the correct hash for zero address:

```suggestion
                    let (mut last_addr, mut hashed_addr) = (Address::ZERO, keccak256(Address::Zero));
```

Reply:
Yeah, thats right, thanks!